#
# Use this patch when upgrading to a new version of Lua.
# To apply the patch, run `git apply patchfile.patch`.
#
diff --git a/vm/src/luaconf.h b/vm/src/luaconf.h
index bdf927e..088200a 100644
--- a/vm/src/luaconf.h
+++ b/vm/src/luaconf.h
@@ -315,12 +315,7 @@
 ** give a warning about it. To avoid these warnings, change to the
 ** default definition.
 */
-#if defined(__GNUC__) && ((__GNUC__*100 + __GNUC_MINOR__) >= 302) && \
-    defined(__ELF__)		/* { */
-#define LUAI_FUNC	__attribute__((visibility("internal"))) extern
-#else				/* }{ */
-#define LUAI_FUNC	extern
-#endif				/* } */
+#define LUAI_FUNC	LUA_API    /* Pallene exports everything */

 #define LUAI_DDEC(dec)	LUAI_FUNC dec
 #define LUAI_DDEF	/* empty */
